### DISCUSSION TITLE
Disable https and change Port

### DISCUSSION THREAD

**Discussion by rene-schwabe** [dunglas/symfony-docker#645] [Q&A] [answered]
Hello,

I would like to a run a symfony on a local network. I need to change the port to "8001" and disable https.

What I tried:
I changed the http-port from 80 to 8001 in the "compose.yaml" and commented out the https ports

It seems the redirect is forced to https. 

How can I change the port and also disable https?

**Comment by 7-zete-7** [comment, marked as answer]
Hello @rene-schwabe.

To disable HTTPS need to set `SERVER_NAME` environment variable to `:80`.

Then forward `80` port from inside as `8001`.

All of this can be done by following command.
 
```shell
SERVER_NAME=:80 HTTP_PORT=8081 docker compose up -d 
```
